unc_classification_tab_data/utils/visualizing_utils.py

```diff
@@ -0,0 +1,211 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from collections import defaultdict
+import itertools
+import unc_classification_tab_data.utils.modeling_utils as gen_utils
+
+
+class ResultContainer:
```

Comment:
For future reference you could look into Dataclasses for this kind of things

Reply:
Ahh yes I will try that later! Could definitely be useful